tests/test_merge.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from itertools import permutations
+from src.mergesort import merge_sort
+
+TO_SORT = (
+    [[]] +
+    [[1]] +
+    list(map(list, permutations(range(4)))) +
+    list(map(list, permutations(range(5))))
+)
```

Comment:
You should probably consider a few more cases. What if the input list has all the same values except for one?  What if there's negative numbers?